src/services/ethereum/transaction.ts

```diff
@@ -58,7 +61,11 @@ export class TransactionExecutor implements ITransactionExecutor {
     if (!hasCorrectNonce.res) {
       return hasCorrectNonce.status!;
     }
-
+    const isSenderNonceGreaterThanScheduled = await this.isSenderNonceGreaterThanScheduled(scheduled);
```

Comment:
Are you sure this is ok? Looking at `hasCorrectNonce` for the condition `senderNonce > scheduled.nonce` it will return `false` so `if` in line `61` will be `true` so the function is gonna `return`. Your function looks at the same exact condition so how is that going to work?